client/client-multi/client-cpp/test/impl/log/LogCollectorTest.cpp

```diff
@@ -41,6 +41,8 @@
 
 namespace kaa {
 
+const std::size_t logDeliveryTime = 1;
```

Comment:
I'd rename it to point out that this variable is nothing but the mock and doesn't affect the tests results.